storybookjs/storybook
# Vite: Update sandbox template for vue2 script

Issue: #20748

## What I did

Update the Vue2/Vite loading script to use the official vue package. 

## How to test

1. Run `yarn task --task dev --start-from=install --template vue2-vite/2.7-js`
2. Open Storybook in your browser
3. Access any story (should not show any import errors)

## Checklist

      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
